ui/src/components/Search.js

```diff
@@ -107,11 +121,9 @@ export default ({ setState, state, setSQON, sqon, savedSetsContext, history, ...
                   return (
                     <Table
                       {...props}
-                      setSelectedTableRows={selectedRows =>
-                        selected.setModels({ models: selectedRows })
-                      }
+                      setSelectedTableRows={selectedRows => selected.setModels(selectedRows)}
                       keepSelectedOnPageChange={true}
-                      initalSelectedTableRows={selected.state.models}
+                      initalSelectedTableRows={selected.state.modelIds}
```

Comment:
after clicking clear in the flyout, the context is updated but the arranger/Table's checkboxes are still checked; I think you need to change arranger/Table to take `selectedTableRows` instead of `initalSelectedTableRows` https://github.com/overture-stack/arranger/blob/master/modules/components/src/DataTable/Table/Table.js#L22  and in `componentDidUpdate` there do a `this.setSelectedTableRows` also

Reply:
Ok cool, I remember this behaviour now ... I swear I thought I had done that but i guess not ...

Reply:
I take it back, this is something else, yeah need to update arranger ...